relay-general/src/pii/processor.rs

```diff
@@ -426,6 +428,36 @@ mod tests {
         assert_annotated_snapshot!(event);
     }
 
+    #[test]
+    fn test_span_default_rules() {
+        let ds_config = DataScrubbingConfig {
```

Comment:
Is `exclude_fields` the vector of safe fields? Should we also test that?

Reply:
we test this in other tests already, I don't think we have to add another test for this here

Reply:
Sounds good then 👍